tests/components/watergate/test_init.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for the Watergate integration init module."""
+
+from unittest.mock import patch
+
+from homeassistant.components.valve import ValveState
+from homeassistant.components.watergate.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from . import init_integration
+from .const import MOCK_WEBHOOK_ID
+
+from tests.common import ANY
+from tests.typing import ClientSessionGenerator
+
+
+async def test_async_setup_entry(
+    hass: HomeAssistant, mock_entry, mock_watergate_client
+) -> None:
+    """Test setting up the Watergate integration."""
+    hass.config.components.add("network")
```

Comment:
```suggestion
```
I've never seen this in any other tests, so would be strange if you actually need this